docs/course_materials/exercises/test_exercises/04_Compound_data_types/test_homework_lists.py

```diff
@@ -0,0 +1,108 @@
+"""
+Automatically check student's homework
+----------------------------------------
+The tests will check:
+ 1. The content of the exercise file
+ 2. The execution and output of the exercise file
+ 3. The output with different inputs
+
+Guidelines:
+- You have to specify the dir Student's exercise files should be placed
+in the same dir as this test.
+- Test names are in the form: test_<exercise>.py
+- Before each test should be the appropriate fixture: @pytest.mark.<topic>
+"""
+import pytest
+
+
+@pytest.mark.lists
+def test_lists(helpers):
+    helpers.set_student_file('lists.py')
+    helpers.expected_pycode = [
+        # comment for Create a list exercise
+        [r'''^[#]+.*((C|c)reate|1)''',
+         'comment for exercise number 1 is missing'],
+        # Create two empty lists list1 and list2 in different ways
+        [r'''^list.*\s*=\s*\[\]''', 'use [] to set a list'],
```

Comment:
The error messages here are helpful for the student, but I'm not sure how much helpful they should be, I rather tell the student what they did wrong and not how they need to fix it

Reply:
Do you mean all the answers in this exercise or only the two above?

Reply:
all the exercise

Reply:
Done